src/app/pages/workspace/workspace-text-window/workspace-text-window.component.html

```diff
@@ -52,6 +39,20 @@
       </p-multiSelect>
     </div>
   </div>
+  <div class="col-sm-5">
+    <label class="col-sm-2 col-form-label">Layer</label>
+    <div class="p-fluid">
+      <p-dropdown #layer="ngModel" [style]="{'height': layerVisibilityHeight + 'px'}"
+        name="layer" appendTo="body" [(ngModel)]="selectedLayer" optionLabel="name"
+        placeholder="Select a layer" [options]="(layers$|async) || []" [filter]="true" [showClear]="true" (onChange)="onChangeLayerSelection($event)"  [required]="false">
```

Comment:
Potresti sostituire il placeholder con "Select a layer to start annotating"?

Reply:
vai fatto